spec/integration/profiles_spec.rb

```diff
@@ -75,12 +75,29 @@
       content_types
       auth_header
 
-      parameter name: :data, in: :body, required: true, schema: {
+      parameter name: :data, in: :body, schema: {
         type: :object,
-        data: {
-          type: :object,
-          properties: {
-            attributes: ref_schema('profile')
+        properties: {
+          type: { type: :string, example: 'profile' },
```

Comment:
:+1: Good catch. I feel like there are so many place like this that are easy to forget to put something.